### ISSUE TITLE
Verification not possible on newly logged in windows desktop session

### ISSUE THREAD

**Issue by JMoVS** [element-hq/element-web#27056]
### Steps to reproduce

1. Login to new Element Desktop on Windows session
2. try to verify with another device
3. an error occurs that the device could not be verified and remains in an untrusted state

#### What did you expect?
verification to work
#### What happened instead?
various error messages resulting in an unverified session

### Operating system

Windows

### Application version

Element version 1.11.58

### Will you send logs?

Yes

**Comment by andzejsp** [comment]
So any progress on this problem? Still experiencing this.. :(

**Comment by gongarher** [comment]
Same problem here. I am unable to verify my pc session. I have tried with multiple web browsers and desktop app. Also, I have tried with different security keys, where the verification process ends successfully but my pc session remains unverified :(. I can see messages from my desktop but cannot send a single message 🙃 Any workaround here? Thanks in advance.

**Comment by andzejsp** [comment]
> Same problem here. I am unable to verify my pc session. I have tried with multiple web browsers and desktop app. Also, I have tried with different security keys, where the verification process ends successfully but my pc session remains unverified :(. I can see messages from my desktop but cannot send a single message 🙃 Any workaround here? Thanks in advance.

yes i found a solution, just use this client instead https://schildi.chat/desktop/

**Comment by gongarher** [comment]
> > Same problem here. I am unable to verify my pc session. I have tried with multiple web browsers and desktop app. Also, I have tried with different security keys, where the verification process ends successfully but my pc session remains unverified :(. I can see messages from my desktop but cannot send a single message 🙃 Any workaround here? Thanks in advance.
> 
> yes i found a solution, just use this client instead https://schildi.chat/desktop/

Thanks! That worked

**Comment by cmedcoff** [comment]
This is still a problem, any updates.  Trying to roll this out to a new org.

Using the security key to verify won't display "looks good" until one uploads twice, and even then after pressing "Continue" the "Verify this session" pop-up/dialog remains.

**Comment by andzejsp** [comment]
> This is still a problem, any updates.  Trying to roll this out to a new org.
> 
> Using the security key to verify won't display "looks good" until one uploads twice, and even then after pressing "Continue" the "Verify this session" pop-up/dialog remains.

Forget about it buddy. They were part of a diddy freak off and now in the middle of lawsuit so aint nobody got time for that... use cinny or schildi app

**Comment by cmedcoff** [comment]
Any updates still experiencing this, but just verify with keys.  One has to upload twice to get the obtain the "Looks good" message, the pressing continue does not dismiss the "Verify this session" prompt/dialog, will upload logs.

**Comment by cmedcoff** [comment]
> > This is still a problem, any updates.  Trying to roll this out to a new org.
> > Using the security key to verify won't display "looks good" until one uploads twice, and even then after pressing "Continue" the "Verify this session" pop-up/dialog remains.
> 
> Forget about it buddy. They were part of a diddy freak off and now in the middle of lawsuit so aint nobody got time for that... use cinny or schildi app

What does that even mean?

**Comment by richvdh** [comment]
Apologies, this seems to have bitrotted.

If people are still having trouble verifying new sessions, please open a new issue, and also send debug logs from within *both* clients.
